mscore/pa.cpp

```diff
@@ -220,7 +220,7 @@ bool Portaudio::start(bool)
 //---------------------------------------------------------
 
 bool Portaudio::stop()
-      {
+      {fromLocal8Bit
```

Comment:
Huh?

Reply:
and sure enough that is where the builds fail: https://travis-ci.org/musescore/MuseScore/jobs/480418687#L2773

Reply:
yes, I'm sorry, I realized that